src/Auth/AuthUtils.php

```diff
@@ -0,0 +1,27 @@
+<?php
```

Comment:
so i take it that in modern PHP you don't need a closing tag for this anymore?  TIL